vllm/entrypoints/openai/rpc/client.py

```diff
@@ -61,7 +64,9 @@ def socket(self):
             socket.connect(self.path)
             yield socket
         finally:
-            socket.close()
+            # linger == 0 means discard unsent messages
+            # when the socket is closed
```

Comment:
Can you just explain why this is needed in the comment?

Reply:
Yes, I've expanded the comment and added a reference to the documentation.